### ISSUE TITLE
Increases hit points for both dogs, cows and oxen.

### ISSUE THREAD

**Issue by Russianstory** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#913]
Well, really, honestly, Animals die from one blow by a shamsher .... well, it's funny to hack to death in this way a hefty OX ... or a cow.

**Comment by augustus8825** [comment]
I agree with this. Hit-and-running enemy workers/oxen is way too easy for something that can cripple your enemy so severely. A tickbox/option in the UCP installer for this would be really cool. This might weaken the Horse Archer meta as well.
